queries.sql

```diff
@@ -41,3 +41,63 @@ SELECT *
 SELECT *
     FROM animals
     WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- MILESTONE 2
+-- Inside a transaction update the animals table by setting the species column to unspecified. 
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+    UPDATE animals SET species = 'unespecified';
+    ROLLBACK;
+COMMIT;
+
```

Comment:
- [ ] Your project is missing the below query, Please make sure to add this transaction in the queries.
![image](https://user-images.githubusercontent.com/83803414/183479488-16cd95f8-5ade-4627-b4d3-362b23c477a0.png)